index.js

```diff
@@ -17,11 +17,11 @@ const findWorkspaceRoot = require('find-yarn-workspace-root');
 module.exports = {
   name: require('./package').name,
   fontawesomeConfig: null,
-  nodeModulesPath: null,
+  _nodeModulesPath: null,
 
   treeForVendor(vendorTree) {
     const iconRollups = []
-    const pathToCore = path.join(this.nodeModulesPath, '@fortawesome', 'fontawesome-svg-core');
+    const pathToCore = path.join(this._nodeModulesPath, '@fortawesome', 'fontawesome-svg-core');
```

Comment:
Using `resolve.sync()` as recommended in deprecation message would look like this:

```js
const pathToCore = require('path').dirname(
  require('resolve').sync('@fortawesome/fontawesome-svg-core')
);
```

Reply:
This is a good change too @jelhan, do you want to go ahead and add it here? Or in a separate PR on this this is merged?

We've already imported resolve so it would probably be `path.dirname(resolve.sync('@fortawesome/fontawesome-svg-core'))`.

Reply:
This would be better in a separate PR. It should address all usages of `nodeModulesPath` in my opinion cause having different strategies would be even worse. Also I'm not aware of the edge cases in which a custom `nodeModulePath` must be provided to `Rollup`. Therefor I don't know how to test that changes manually.